hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientResult.java

```diff
@@ -84,6 +85,12 @@ public SqlRowMetadata getRowMetadata() {
         }
     }
 
+    @Nonnull
+    @Override
+    public SqlResultType getResultType() {
+        return SqlResultType.ROWS;
```

Comment:
The result type should be integrated into the client protocol. Currently, it is ignored.

Reply:
fixed